### DISCUSSION TITLE
Can anyone run Marvelous - Mouhitotsu no Takarajima?

### DISCUSSION THREAD

**Discussion by PaulLuigi** [eduardofilo/RG350_adam_image#242] [General]
Hello, I've heard Marvelous - Mouhitotsu no Takarajima is a really cool game made by a Zelda developer that never made it to the US.  I'm trying to run it using Adam 1.3 on an Anbernic RG280v, but so far I haven't had any luck with any of the 3 emus.  I got it to run just one time with the default (retroarch 2005) without sound and what seemed like a low frame rate, but after trying dozens more times, I was never able to get it to launch to the title screen again.  I can still load the save state that I created from that one successful launch, but I was wondering if anyone has had any more success using either a particular rom or particular emulator settings.  Sorry in advance if this is the wrong place to post this.  Thank you!

**Comment by neilswann80** [comment]
I'm getting exactly the same symptoms running it on my RG350M on Adam 1.4.2.  Looks like this will be a tricky game to emulate on mainline emulators, let alone those ported to OpenDingux.  

FYI what I experienced...
- it worked for me once via "RETROARCH_RG350_SFC_S9X2005PLUS.OPK"
- Like you, I created a save, which can be reloaded, but you cannot start the game a second time without a save.
- No sound.
- The other two SNES emulators didn't work at all.

I seem to have intermittent luck getting the game to start again after renaming the rom file, adding a "1" before the .sfc for example.  But again, no sound.
